src/monitor/http.go

```diff
@@ -0,0 +1,31 @@
+package monitor
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/boltdb/bolt"
+	"github.com/rs/cors"
+)
+
+var database *bolt.DB
+
+func LaunchServer(port string) {
+	var err error
+	database, err = bolt.Open("test.db", 0600, nil)
```

Comment:
Why open this db?

Reply:
i create instance of db for all requests. Or you mean why do i open "test.db"?

Reply:
yes, should not be opening, should not have global `database`